shakedown/dcos/master.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for working with master"""
+
+from shakedown import *
+
+
+def partition_master(incoming=True, outgoing=True):
+    """ Partition Master's port alone. To keep DC/OS cluster running.
+    """
+
+    print('Partitioning master. Incoming:{} | Outgoing:{}'
+        .format(incoming, outgoing))
+
+    if incoming and outgoing:
+        copy_file_to_master("{}/partition_master_cmd"
+            .format(shakedown_dcos_dir()))
+        run_command_on_master("sh partition_master_cmd")
```

Comment:
Is there a reason that you're copying a bash script to a node only to execute it? Can't these commands be executed from the machine running Shakedown, avoiding the bash script altogether?